cmd/portgrpc/wire.go

```diff
@@ -29,6 +29,8 @@ func BuildDIForApp() (*App, error) {
 		// usecase:
 		usecases.NewCreatePort,
 		wire.Bind(new(usecasefacades.PortCreator), new(*usecases.CreatePort)),
+		usecases.NewGetPorts,
+		wire.Bind(new(usecasefacades.PortGetter), new(*usecases.GetPorts)),
```

Comment:
Think about this: ports grpc now need GetPorts usecase because of the way facade interface is defined